src/utils/py/daos_api.py

```diff
@@ -998,10 +1017,11 @@ def single_insert(self, dkey, akey, value, size, tx):
             self.iod.iod_type = 1
             self.iod.iod_size = size
             self.iod.iod_nr = 1
-            #self.iod.iod_eprs = ctypes.cast(ctypes.pointer(self.epoch_range), ctypes.c_void_p)
-            iod_ptr = ctypes.pointer(self.iod)
+            #self.iod.iod_eprs = ctypes.cast(ctypes.pointer(self.epoch_range),
+            #                                ctypes.c_void_p)
+            _iod_ptr = ctypes.pointer(self.iod)
```

Comment:
(pylint-unused-variable) Unused variable '_iod_ptr'

Reply:
Why are we setting `_iod_ptr` if it's not used?